orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/EvaluateDeploymentHealthTaskSpec.groovy

```diff
@@ -198,6 +204,202 @@ class EvaluateDeploymentHealthTaskSpec extends Specification {
     false              | null             || ExecutionStatus.FAILED_CONTINUE
   }
 
+  def "should handle retrofit http error and return the task status depending on the scenarios"() {
```

Comment:
I'm not sure I see the point of all this added test coverage.  I mean, I'm all for more test coverage, but if I'm reading the code correctly, [EvaluateDeploymentHealthTask](https://github.com/spinnaker/orca/blob/5009613320ad9a0b0765a1c6c9d1f567dd1f0be4/orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/EvaluateDeploymentHealthTask.java) doesn't catch any RetrofitErrors (nor have any catch blocks at all), so I don't see how adding SpinnakerRetrofitErrorHandler to DeploymentMonitorService (e.g. in https://github.com/spinnaker/orca/pull/4614) could change behavior specifically in EvaluateDeploymentHealthTask,

Reply:
@dbyron-sf - This is exactly what I mentioned in the comments : https://github.com/spinnaker/orca/pull/4614#discussion_r1422295158

Reply:
The easiest way to resolve that thread is with some git juggling to move the addition of that one specific test case to a separate commit before the code change in that PR.  I suppose these additional tests could be related, but it's hard to see how.

Reply:
These additional test cases are similar, only differences are the type of Spinnaker*Exceptions are different

Reply:
I'd rather focus on https://github.com/spinnaker/orca/pull/4614.

Reply:
Added test case for EvaluateDeploymentTask and moved to a separate commit. : d481b847e28482e16b9fa94e83c1a917e85c00d2